jwst/datamodels/schemas/fgsimg_vegamag_offset.schema.yaml

```diff
@@ -0,0 +1,17 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/fgsimg_vegamag.schema"
+title: FGS Vega magnitude offset data model
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_exptype.schema
+- $ref: keyword_pexptype.schema
+- type: object
+  properties:
+    vegamag_offset_table:
+      title: Vega magnitude offsets from AB magnitudes table
+      fits_hdu: VEGAMAG_OFFSET
```

Comment:
Arrgh, these are going to be FITS tables?  How gross. ASDF would be so much nicer.

Reply:
Is there a specific reason they have to be FITS tables? ASDF would make for a files that is easier to understand and maintain.

Reply:
No, I was just following the `photom` and `apcorr` examples, which are both FITS.  I can make them ASDF, although the INS teams seem to prefer FITS tables.

Reply:
ASDF is so much simpler and more appropriate for simple table-based data like this. We already have a number of reference files in ASDF format (e.g. some of the WCS-related ref files), so there is precedence. Since you're the one creating them, I'd suggest using ASDF, which would also then help others to start down the ASDF road. We suggested ASDF for the apcorr ref files, but there seemed to be an assumption that a new format would take time to learn and time was of the essence (at least back then) to get them created and delivered.

Reply:
We also have the `filteroffset` reference file type which may serve as an example.

Reply:
Updated to use ASDF.